scriptworker/constants.py

```diff
@@ -167,6 +169,8 @@
 
         "mobile-1-images",  # there is no mobile level 2.
         "mobile-3-images",
+        # TODO: to change this to something else?
```

Comment:
I agree to change this to something like `services-X-decision`, or `services-X-images`, in a followup